truetype/truetype.go

```diff
@@ -306,6 +317,65 @@ func (f *Font) parseMaxp() error {
 	return nil
 }
 
+func (f *Font) parseName() error {
+	if len(f.name) < 18 {
+		return FormatError("name data too short")
+	}
+	if u16(f.name, 0) != 0 {
+		return FormatError(fmt.Sprintf("invalid format specifier: %d", u16(f.name, 0)))
+	}
+	numNames := int(u16(f.name, 2))
+	stringOffset := u16(f.name, 4)
+	font := &nr{}
+
+	for offset, i := 6, 0; i < numNames; i, offset = i+1, offset+12 {
+		nameID := u16(f.name, offset+6)
+		switch nameID {
+		case 1, 2:
+			/* Keeping these here in case they're requested later.
+
+			platformSpecificID, languageID,platformID := u16(f.name, offset+2), u16(f.name, offset+4),u16(f.name, offset)
+			*/
+			length, nameOffset := u16(f.name, offset+8), u16(f.name, offset+10)
+
+			// Assume a non-UTF-16 value
+			value := f.name[stringOffset+nameOffset : stringOffset+nameOffset+length]
+
+			// but just in case... if the first byte is 0, as in luxi's 13th nameRecord,
+			// assume UTF-16, try to parse it if its length is even
+			if value[0] == 0 && length&1 == 0 {
+				var err error
+
+				enc := unicode.UTF16(unicode.BigEndian, unicode.IgnoreBOM)
```

Comment:
The [spec](https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6name.html) says:

```
In particular, Unicode names should be in UTF-16 (i.e., two-byte characters). UTF-8 and UTF-32 should not be used.
```

....but that's for Unicode fonts.

`luxisr.ttf` identifies itself with the Macintosh platformID and Roman platformSpecificID.  Apple's developer site doesn't specify whether they use UCS-2 or UTF-16.